src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,24 @@
+import { createSlice } from '@reduxjs/toolkit';
+import books from '../../data/books.json';
+
+const initialState = {
+  books,
+};
```

Comment:
As required, the initial state should be an array of books that is initially empty😉.
<img width="472" alt="image" src="https://github.com/henokkhm/microverse-bookstore-app/assets/64643995/b0d80a53-13ae-47d2-b602-d356cf115a04">